docs/guides/configure-error-mitigation.mdx

```diff
@@ -162,8 +162,8 @@ options.twirling.enable_gates = True
 # Turn on measurement error mitigation.
 options.resilience.measure_mitigation = True
 
-print(f">>> gate twirling is turned on: {estimator.options.twirling.enable_gates}")
-print(f">>> measurement error mitigation is turned on: {estimator.options.resilience.measure_mitigation}")
```

Comment:
It looks like you've removed the entire line? You should just be removing the `>>>`